lib/http-bridge/test/bench/Main.hs

```diff
@@ -216,7 +222,7 @@ bench_restoration _ (wid, wname, s) = withHttpBridge network $ \port -> do
     w <- newWalletLayer @_ @t nullTracer bp db nw tl
```

Comment:
Opted to preserve the previous behaviour by keeping the `nullTracer` for the WalletLayer.